k8s/chainctl

```diff
@@ -265,12 +284,7 @@ case "$1" in
     ;;
 
   patch)
-    if [ -z "$2" ] ; then
-      printf "Error: no deployment parameter supplied.\n\n"
-      usage
-      exit 1
-    fi
-    deployment="$2"
+    deployment=`kubectl get deploy | grep -v NAME | awk '{print$1}'`
```

Comment:
`chainctl` is designed to support multiple deploys, so this would patch every deploy rather than just a single one, that might not be what is intended, but I haven't used this command before so I'm not sure?

Reply:
Right, I thought about that too, and what I came up with is now since we have 1 cluster=1 deploy, we're pretty much good with taking the only one returned.

Reply:
If that's the case, then I'd prefer to remove support for multiple targets from `chainctl` rather than half support them.

Though we are just looking into adding a deploy with multiple targets to do CI on!

Reply:
Maybe I don't understand what we would be half supporting. If you're on a Chainlink cluster, each command works like normal, same as if you were on an Ethereum cluster.